sql/src/test/java/io/crate/planner/InsertPlannerTest.java

```diff
@@ -94,7 +94,7 @@ public void prepare() throws IOException {
     }
 
     @Test
-    public void testInsertFromSubQueryNonDistributedGroupBy() {
```

Comment:
in the test names we still want to keep the knowledge of what we are testing. imho, `testInsertFromSubQueryNonDistributedGroupBy` was alright,

Reply:
I was not finished yet with this pr, currently going through all the testmethods.

Reply:
But yes, you are right ;-)

Reply:
oops, sorry :)

Reply:
So I renamed now only these testcases where subqueries are actually involved.